testing/MLDB-621_serial_procedure_deadlock.py

```diff
@@ -1,16 +0,0 @@
-#
-# MLDB-621_serial_procedure_deadlock.py
```

Comment:
Why removing it?  Isn't still detecting that recursive references aren't deadlocking?